Can't compile on recent kernel

When I try to install the module on a 5.3.0 linux kernel, I get his error
```
./src/rawchannel.cc:582:38: error: ‘SIOCGSTAMP’ was not declared in this scope; did you mean ‘SIOCGARP’?
  582 |         if (likely(ioctl(m_SocketFd, SIOCGSTAMP, &tv) >= 0))
```
It seems it comes from this change in the kernel :
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=0768e17073dc

--- a/src/rawchannel.cc
+++ b/src/rawchannel.cc
@@ -38,6 +38,7 @@
 
 #include <linux/can.h>
 #include <linux/can/raw.h>
+#include <linux/sockios.h>
 
 #include <vector>
 #include <string>

